strider/trapi.py

```diff
@@ -89,17 +89,21 @@ def merge_edges(kedges: list[Edge]) -> Edge:
 def merge_kgraphs(kgraphs: list[KnowledgeGraph]) -> KnowledgeGraph:
     """ Merge knowledge graphs. """
 
+    if len(kgraphs) == 1:
+        return {
+            "nodes": kgraphs[0]["nodes"],
+            "edges": kgraphs[0]["edges"],
+        }
+    assert len(kgraphs) == 2
```

Comment:
I don't really understand why this function is limited to merging two kgraphs, or why it returns early. Can you enlighten me?

Reply:
I left in statements after the return in order to hint to future developers that this was really meant as a temporary change until we find a less lossy solution to the merging speed problem. We merge only two kgraphs at a time because, in my limited experiments, this was the speediest solution. dict unpacking cannot be used in dict comprehension, or I'd have made it more general.